specification.md

```diff
@@ -1,6 +1,6 @@
 # WCIF
 
-*Version: 1.0*
+*Version: 1.1*
```

Comment:
Actually, this doesn't change the spec per se, because it already allowed arbitrary roles to be included. Now it only outlines another role that we consider "standard", so I think it's fine to keep the version. @gregorbg let me know if you have any objections :)

```suggestion
*Version: 1.0*
```

FTR we haven't bumped the version so far, because during the initial evolution of the spec it was in a sort of "beta" state. As things settled we crated this repo, and there hasn't been much changes since. We can start bumping the version if there's a change with clear implications for existing apps, in which case we should also keep a changelog.

Reply:
No hard feelings, we can just leave the version at 1.0 in this commit. But shouldn't we generally bump it because of the changes to qualifications?

Reply:
Theoretically yes, but qualification is not implemented on the WCA website, so I cannot think of any tool actually using it. In wca-live it was relevant only because we destructure the whole WCIF on import, but even there it doesn't matter since so far we were only importing `null` ^^